src/components/SourceTabs.js

```diff
@@ -124,12 +146,25 @@ const SourceTabs = React.createClass({
       click: () => showSource(tab)
     };
 
+    const copySourceUrl = {
+      id: "node-menu-close-tabs-to-right",
+      label: "Copy Link Address",
+      accesskey: "X",
+      disabled: false,
+      click: () => {
+        showSource(tab);
+        this.copyToTheClipboard();
```

Comment:
Hi @jasonLaster 

<img width="192" alt="screen shot 2016-12-09 at 4 43 28 pm" src="https://cloud.githubusercontent.com/assets/4562118/21042761/7463cede-be2e-11e6-8895-b4a368f9924d.png">

Since `tab` equals to a number, I use `showSource` to retrieve this tab's url.